network-store-server/src/main/java/com/powsybl/network/store/server/migration/V211LimitsQueryCatalog.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.network.store.server.migration;
+
+import static com.powsybl.network.store.server.QueryCatalog.*;
+
+/**
+ * @author Etienne Homer <etienne.homer at rte-france.com>
+ */
+//Class to be deprecated when limits are fully migrated - should be after v2.13 deployment
```

Comment:
remove comment, we keep this for the migration for eternity

Reply:
done